app/services/stats/management_information_generator.rb

```diff
@@ -10,6 +10,7 @@ def self.call(options = {})
 
     def initialize(options = {})
       @format = options.fetch(:format, DEFAULT_FORMAT)
+      @claim_scope = options.fetch(:claim_scope, :all)
```

Comment:
(nit, and possibly not really part of this PR) Unless there are lots of arguments I would prefer;

```ruby
def initialize(format: 'csv', claim_scope: :all)
  @format = format # And the DEFAULT_FORMAT constant can be removed
  @claim_scope = claim_scope
end
```

(Also `:csv` instead of `'csv'` for the default format but that may have a knock-on effect)

Reply:
Unless we change `StatsReportGenerator.initialize` to match this would require its call to be amended as below, which just shifts the yuckiness. options hashes are a pretty common ruby idiom, albeit increasingly out of fashion for kwargs.

```
# in Stats::StatsReportGenerator
ManagementInformationGenerator.call(format: options[:format], claim_scope: options[:claims_scope])
```

And amending the generalised `Stats::StatsReportGenerator` as below does not feel right to me at the moment as claim_scope is only applicable to MI reports.
```
module Stats
  class StatsReportGenerator
  ...
    def initialize(report_type, format: 'csv', claims_scope: :all)
      @report_type = report_type
      @options = options
    end
    ...
   end
end
```

The next tranch of work may involve further refactoring which could smooth this out.

Reply:
OK. No problem.